webcomponents/tool/src/main/frontend/js/sakai-search.js

```diff
@@ -90,51 +90,50 @@ class SakaiSearch extends SakaiElement {
   render() {
 
     return html`
-    <div class="input-group position-relative" role="search">
-    <input type="search"
+    <form class="input-group position-relative" onsubmit="return false">
+      <input type="search"
         class="sakaiSearch form-control"
         autocomplete="off"
         id="sakai-search-input"
+        pattern=".{3,}" title="${this.i18n.search_min_length}"
```

Comment:
It would be nice if the length "3" could be a variable/property. I think currently the minimum is "2" right? I would suggest to do that for the resource properties too using a placeholer like `{}` then you use it like `${this.i18n.search_min_length.replace("{}", minInputLength)}`. That would improve maintainability. and for the pattern `pattern=".{${minInputLength},}"`

Reply:
3 is the correct length value, I like the idea of keeping the attribute pattern as a variable. Thanks for the suggestion, Markus :+1: )